package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "publish",
```

Comment:
We will need to pick a more fully-qualified name here, since, unsurprisingly, the name "publish" is already taken in the npm registry - https://www.npmjs.com/package/publish
Perhaps something like fluid-publishdev? (capital letters are disallowed in npm package names)